docs/components/WizardLayoutView.jsx

```diff
@@ -155,7 +176,7 @@ export default class WizardLayoutView extends React.PureComponent {
   }
 
   _renderConfig() {
```

Comment:
Can we add a button/link here that navigates to `/fullscreen-wizard-layout`? That would be nice

Reply:
Good call. Was thinking of linking to it somewhere but couldn't think of where the best spot would be. This seems solid.